websites/A/Audible/presence.ts

```diff
@@ -0,0 +1,56 @@
+const presence = new Presence({
+		clientId: "991933219719086080",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+		startTimestamp: browsingTimestamp,
+	};
+	if (document.location.pathname.includes("/account"))
+		presenceData.details = "Viewing account";
+	else if (document.location.pathname.includes("/wl"))
+		presenceData.details = "Viewing wishlist";
+	else if (document.location.pathname.includes("/library"))
+		presenceData.details = "Viewing library";
+	else if (document.location.pathname.includes("plus"))
+		presenceData.details = "Viewing plus catalog";
+	else if (document.location.pathname.includes("gift"))
+		presenceData.details = "Viewing gift center";
+	else if (document.location.pathname.includes("/pd")) {
+		const title = document.querySelector(
+				"h1.bc-heading.bc-color-base.bc-pub-break-word.bc-text-bold"
+			).textContent,
+			cover = document
+				.querySelector("img.bc-pub-block.bc-image-inset-border.js-only-element")
+				.getAttribute("src");
+		presenceData.details = `Viewing ${title} by ${
+			document.querySelector(
+				"li.bc-list-item.authorLabel > a.bc-link.bc-color-link"
+			).textContent
+		}`;
+		presenceData.largeImageKey = cover;
```

Comment:
Apply the same logic from here https://github.com/PreMiD/Presences/pull/6408/files#r917325055